### ISSUE TITLE
Sitemap with nuxt content fails to render

### ISSUE THREAD

**Issue by BobTheShoplifter** [harlan-zw/nuxt-simple-sitemap#31]
### Describe the bug

Using the addon with nuxt content results in a rendering error and boat load of random pages.

Adding the sitemap addon to the official Nuxt Content stackblitz https://stackblitz.com/edit/github-9layqu

(Based on the sitemap it seems to be <!DOCTYPE included in the sitemap)

![image](https://user-images.githubusercontent.com/22559547/231177362-19be9dfe-ca81-4f8d-96ec-9032a526cab6.png)

### System / Nuxt Info

```XML
<?xml version="1.0" encoding="UTF-8"?><?xml-stylesheet type="text/xsl" href="/__sitemap__/style.xsl"?>
<urlset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xhtml="http://www.w3.org/1999/xhtml" xmlns:image="http://www.google.com/schemas/sitemap-image/1.1" xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd http://www.google.com/schemas/sitemap-image/1.1 http://www.google.com/schemas/sitemap-image/1.1/sitemap-image.xsd" xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
    <url>
        <loc>http://localhost:3000/0</loc>
    </url>
    <url>
        <loc>http://localhost:3000/1</loc>
    </url>
    <url>
        <loc>http://localhost:3000/2</loc>
    </url>
    <url>
        <loc>http://localhost:3000/3</loc>
    </url>
    <url>
        <loc>http://localhost:3000/4</loc>
    </url>
    <url>
        <loc>http://localhost:3000/5</loc>
    </url>
    <url>
        <loc>http://localhost:3000/6</loc>
    </url>
    <url>
        <loc>http://localhost:3000/7</loc>
    </url>
    <url>
        <loc>http://localhost:3000/8</loc>
    </url>
    <url>
        <loc>http://localhost:3000/9</loc>
    </url>
    <url>
        <loc>http://localhost:3000</loc>
    </url>
    <url>
        <loc>http://localhost:3000/!</loc>
    </url>
    <url>
        <loc>http://localhost:3000/D</loc>
    </url>
    <url>
        <loc>http://localhost:3000/O</loc>
    </url>
    <url>
        <loc>http://localhost:3000/C</loc>
    </url>
    <url>
        <loc>http://localhost:3000/T</loc>
    </url>
    <url>
        <loc>http://localhost:3000/Y</loc>
    </url>
    <url>
        <loc>http://localhost:3000/P</loc>
    </url>
    <url>
        <loc>http://localhost:3000/E</loc>
    </url>
    <url>
        <loc>http://localhost:3000/h</loc>
    </url>
    <url>
        <loc>http://localhost:3000/t</loc>
    </url>
    <url>
        <loc>http://localhost:3000/m</loc>
    </url>
    <url>
        <loc>http://localhost:3000/l</loc>
    </url>
    <url>
        <loc>http://localhost:3000/e</loc>
    </url>
    <url>
        <loc>http://localhost:3000/a</loc>
    </url>
    <url>
        <loc>http://localhost:3000/d</loc>
    </url>
    <url>
        <loc>http://localhost:3000/c</loc>
    </url>
    <url>
        <loc>http://localhost:3000/r</loc>
    </url>
    <url>
        <loc>http://localhost:3000/s</loc>
    </url>
    <url>
        <loc>http://localhost:3000/=</loc>
    </url>
.....
<!-- XML Sitemap generated by Nuxt Simple Sitemap -->
```

**Comment by harlan-zw** [comment]
Looks like a bug with catch-all routes `[...slug].vue`

I'd recommend using `documentDriven` mode if you want behavior like this, otherwise, I'll try and fix it soon.

**Comment by harlan-zw** [comment]
Hey @BobTheShoplifter, this is fixed in v2.4.9
